src/response/response.ts

```diff
@@ -0,0 +1,43 @@
+import { MessageReply } from './message-reply';
+
+type ResponseStatus = {
+  code: number
+  message: string
+};
+
+type ResponseOpts = {
+  status?: ResponseStatus
+  results?: MessageReply[];
+};
+
+/**
+ * class used to create a response that will be returned for a given request
+ */
+export class Response {
+  // present ONLY if there is a general request-related issue
+  // e.g. malformed request, invalid target. `status` and `replies` are mutually exclusive
+  status?: ResponseStatus;
+  // responses to individual messages provided within a request
+  results?: MessageReply[];
```

Comment:
`replies` & same for rest of "results"

Reply:
done: https://github.com/TBD54566975/dwn-sdk-js/commit/7d86edbe7891f4f0fb582d44c721acc289660d0b